src/integration/mod.rs

```diff
@@ -826,3 +849,25 @@ impl BlockBuilderCreatorValidator {
         Ok(None)
     }
 }
+
+fn create_deposit_tx() -> Bytes {
+    const ISTHMUS_DATA: &[u8] = &hex!(
+        "098999be00000558000c5fc500000000000000030000000067a9f765000000000000002900000000000000000000000000000000000000000000000000000000006a6d09000000000000000000000000000000000000000000000000000000000000000172fcc8e8886636bdbe96ba0e4baab67ea7e7811633f52b52e8cf7a5123213b6f000000000000000000000000d3f2c5afb2d76f5579f326b0cd7da5f5a4126c3500004e2000000000000001f4"
+    );
+
+    let deposit_tx = TxDeposit {
+        source_hash: B256::default(),
+        from: address!("DeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001"),
+        to: TxKind::Call(address!("4200000000000000000000000000000000000015")),
+        mint: None,
+        value: U256::default(),
+        gas_limit: 210000,
+        is_system_transaction: true,
```

Comment:
does this deposit tx work? had to set is_system_transaction: false for it to work in op-rbuilder

Reply:
Our integration tests are running in an old (two weeks) version of Reth